MaxLairInstance.py

```diff
@@ -220,11 +222,27 @@ def identify_pokemon(self,
         for pokemon in self.rental_pokemon.values():
             # Build the composite identifying string with the same format as the OCRed text
             #   Note that some OCR strings omit the ability and others omit the types so don't include these identifiers in these cases
-            string_to_match = pokemon.name.split(' (')[0]
+            if language == 'English':
+                string_to_match = pokemon.name.split(' (')[0]
+            if language == 'Spanish':#for now spanish is using english

```

Comment:
I haven't used the spanish file yet. I'd rather wait for miguel to test if my changes work in spanish. Otherwise, I'm gonna break the script for him